test/src/note/frequency_test.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:music_notes/music_notes.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('.isHumanAudible', () {
```

Comment:
Please, add a new parent group for the tested class in this file. Although redundant, it helps debug the context of each test from logs.

```suggestion
  group('Frequency', () {
    group('.isHumanAudible', () {
```